iam/api-client/custom_roles_test.py

```diff
@@ -33,22 +33,23 @@
 
 @pytest.fixture(scope="module")
 def test_custom_role():
-    # section to create custom role to test policy updates.
-    custom_roles.create_role(
-        CUSTOM_ROLE_NAME,
-        GCLOUD_PROJECT,
-        CUSTOM_ROLE_TITLE,
-        CUSTOM_ROLE_DESCRIPTION,
-        CUSTOM_ROLE_PERMISSIONS,
-        CUSTOM_ROLE_STAGE,
-    )
-    yield CUSTOM_ROLE_NAME
-
-    # Delete the custom role
+    # This custom role is reused in read/update tests.
     try:
-        custom_roles.delete_role(CUSTOM_ROLE_NAME, GCLOUD_PROJECT)
-    except googleapiclient.errors.HttpError:
-        print("Custom role already deleted.")
+        custom_roles.create_role(
+            CUSTOM_ROLE_NAME,
+            GCLOUD_PROJECT,
+            CUSTOM_ROLE_TITLE,
+            CUSTOM_ROLE_DESCRIPTION,
+            CUSTOM_ROLE_PERMISSIONS,
+            CUSTOM_ROLE_STAGE,
+        )
+    except googleapiclient.errors.HttpError as e:
+        if "HttpError 409" not in str(e):
+            raise e
+        # Ignore error since we just reuse the same custom role.
+        pass
```

Comment:
Instead of a pass, please make this an informational print statement

Reply:
It's almost always not printed, but I added it :)